test/HomeControllerTest.java

```diff
@@ -27,24 +25,69 @@
  * If you are interested in mocking a whole application, see the wiki for more details.
  *
  */
-public class HomeControllerTest {
+public class HomeControllerTest extends WithApplication{
+
+    private String apprenticeCSV;
+    private String scheduleCSV;
+    private List<FridayLunch> loadedSchedule;
+
+    @Override
+    protected Application provideApplication() {
+        return new GuiceApplicationBuilder()
+                .configure("play.http.router", "router.Routes")
+                .build();
+    }
+
+    @Before
+    public void setUp() throws IOException {
+        apprenticeCSV = "/Users/molliestephenson/Java/LunchMan/test/mockApprentices.csv";
+        scheduleCSV = "/Users/molliestephenson/Java/LunchMan/test/mockSchedule.csv";
+        loadedSchedule = createScheduleFromCSV(new ArrayList<>(), scheduleCSV);
+
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        CSVHelper.saveRotaToCSV(loadedSchedule, scheduleCSV);
+    }
 
     @Test
     public void indexPage() {
-        HomeController homeController = new HomeController();
+        HomeController homeController = new HomeController(apprenticeCSV, scheduleCSV);
         Result result = homeController.index();
         assertTrue(contentAsString(result).contains("LunchMan"));
     }
 
     @Test
     public void indexPageShowsApprenticeNames() {
-        HomeController homeController = new HomeController();
+        HomeController homeController = new HomeController(apprenticeCSV, scheduleCSV);
         Result result = homeController.index();
         assertTrue(contentAsString(result).contains("Priya"));
         assertTrue(contentAsString(result).contains("Mollie"));
         assertTrue(contentAsString(result).contains("Nick"));
         assertTrue(contentAsString(result).contains("Rabea"));
-        assertTrue(contentAsString(result).contains("Ced"));
     }
-    
+
+    @Test
+    public void canChangeShownApprenticeForADay() throws Exception {
```

Comment:
again, might be nice to layout the tests with the odd new line here and there, to break up the act, arrange, assert.
Its a matter of personal preference, but when faced with a test that looks as dense as this,  its hard to see at a glance what the method under test is.